pyproject.toml

```diff
@@ -27,11 +27,10 @@ folders = [{ path = "auth0" }]
 
 [tool.poetry.dependencies]
 python = ">=3.8"
-aiohttp = "^3.8.5"
-cryptography = "^42.0.4" # pyjwt has a weak dependency on cryptography
-pyjwt = "^2.8.0"
-requests = "^2.31.0"
-urllib3 = "^2.0.7"       # requests has a weak dependency on urllib3
```

Comment:
`urrlib3` is removed since it is not used directly by auth0-python. `requests` has an explicit dependency on `urrlib3`: https://github.com/psf/requests/blob/main/setup.cfg#L6